docs/migrate/migrate-e2e.rst

```diff
@@ -0,0 +1,258 @@
+
+
+   sampler = Sampler(options={"shots": shots})
+   ae = IterativeAmplitudeEstimation(epsilon, alpha=alpha, sampler=sampler)
+
+
+   result = ae.estimate(problem)
+   print(result)
+
+
+   {   'alpha': 0.05,
+       'circuit_results': None,
+       'confidence_interval': (0.299362359799064, 0.30061213376177465),
+       'confidence_interval_processed': (0.299362359799064, 0.30061213376177465),
+       'epsilon_estimated': 0.0006248869813553215,
+       'epsilon_estimated_processed': 0.0006248869813553215,
+       'epsilon_target': None,
+       'estimate_intervals': [   [0.0, 1.0],
+                                 [0.2828417741377001, 0.3216648682822661],
+                                 [0.299362359799064, 0.30061213376177465]],
+       'estimation': 0.2999872467804193,
+       'estimation_processed': 0.2999872467804193,
+       'num_oracle_queries': 61440,
+       'powers': [0, 0, 15],
+       'ratios': [1.0, 31.0],
+       'shots': None,
+       'theta_intervals': [   [0, 0.25],
+                              [0.08924749081234563, 0.09597799202522334],
+                              [0.09214176853962333, 0.09235879737893671]]}
+
+
+
+Code description
+----------------
+
+The code with primitives assumes that the user is running their code
+locally, hence the reference to import
+``from qiskit.primitives import Sampler``. After the algorithm is
+adjusted to use a primitive, we initialize the primitive and then pass
+it to the algorithm.
+
+Step 1. Import the correct primitive package
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+How you import the primitive depends on what type of backend you want to run on.  
+
+* If you want to use **IBM Cloud resources** for simulating quantum circuits classically or running them on real quantum hardware, run ``from qiskit_ibm_runtime import Sampler``    ``from qiskit_ibm_runtime import QiskitRuntimeService``.
+* If you don't need a lot of advanced features and **just want to run tests**, run ``from qiskit.primitives import Sampler``. This allows for an exact or shot-based classical simulation of quantum circuits. For the shot-based case, a normal probability distribution is fixed.
+* If you want to run **advanced classical simulations** of quantum algorithms, run ``from qiskit.providers.aer.primitives import Sampler``.  This gives you access to an array of quantum circuit classical simulators that are better optimized and more customizable than the previous option.
+
+.. note::
+   
+   Similar import options exist for the Estimator primitive. 
+
+Step 2. Create the primitive instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To use default settings we use the following code:
+
+``sampler = Sampler()``
+
+We can modify run options which are passed to the primitive as
+a Python dictionary. For example, setting the number of shots, which can
+be done as follows:
+
+.. code-block:: python
+
+   options = {"shots": 1024}
```

Comment:
This doesn't work with runtime primitives. Qiskit Runtime primitives have more options, so the options are organized in categories. So to specify shots one would instead do

```
options  = {"execution": {"shots": 1024}}
```
(yes we should change it to make things easier for users)